### ISSUE TITLE
how about using multithread for test?

### ISSUE THREAD

**Issue by ParkHanbum** [namhyung/uftrace#257]
current test case over 170. too long to test. 
so, I suggest using multithread for test!

**Comment by namhyung** [comment]
It'd be nice to have.  But some test cases can affect others (by using same intermediate file or changing global settings).

**Comment by honggyukim** [comment]
Why don't we change different intermediate files for each optimization and cflags(-pg or -finstrument-functions), then run and test them in parallel? I think we don't have to run different tests in parallel.

**Comment by namhyung** [comment]
Probably we can run some tests in parallel, but at least kernel tests should be run serially.

**Comment by honggyukim** [comment]
I think it's good to be done in the future but not in the current work item so why don't we close it? We can reopen it when it's ready to be done.

**Comment by namhyung** [comment]
I don't want to close unfinished work.  Once closed, I won't take a look at it again. ;-p

**Comment by namhyung** [comment]
Merged: #570
